lib/sensu-plugins-disk-checks/version.rb

```diff
@@ -2,7 +2,7 @@ module SensuPluginsDiskChecks
   module Version
     MAJOR = 3
     MINOR = 1
-    PATCH = 0
+    PATCH = 1
```

Comment:
In general the versioning (and dating in the changelog) is left to the maintainers for several reasons:
- we may disagree on the impact of the change and therefore how its versioned
- we date the release rather then when the contribution was submitted, PR review and acceptance has no SLA and is not guaranteed to be merged within the same day
- PRs are not a FIFO queue, as such versioning them prior to acceptance leads to needing to ask people to rebase more often

Reply:
hey sorry for that, yes, totally, that was just so I could deploy a fix to my internal systems!, sorry was never meant to be part of this PR!

Reply:
No worries